drivers/all_neighbors_system_dashboard/app/views/all_neighbors_system_dashboard/warehouse_reports/reports/raw.haml

```diff
@@ -0,0 +1,5 @@
+- # FIXME: need to bring in additional CSS
```

Comment:
Are these notes all still relevant?

Reply:
At the moment yes.  Fixing now, hopefully.